cabal.project

```diff
@@ -77,3 +69,35 @@ allow-newer: servant-multipart-api:servant
 
 -- ghc 9
 allow-newer: tdigest:base
+
+-- ghc 9.2
+allow-newer: basement:base
+allow-newer: binary:bytestring
+allow-newer: generics-sop:base
+allow-newer: generics-sop:ghc-prim
+allow-newer: generics-sop:template-haskell
+allow-newer: http-media:base
+allow-newer: lucid:base
+allow-newer: memory:base
+allow-newer: openssl-streams:bytestring
+allow-newer: servant-js:base
+allow-newer: servant-js:base-compat
+allow-newer: servant-js:lens
+allow-newer: servant-multipart:bytestring
+allow-newer: servant-multipart:bytestring
+allow-newer: servant-multipart:http-media
+allow-newer: servant-multipart:lens
+allow-newer: servant-multipart:resourcet
+allow-newer: servant-swagger:base
+allow-newer: servant-swagger:base-compat
+allow-newer: servant-swagger:base-compat
+allow-newer: servant-swagger:Cabal
+allow-newer: singleton-bool:base
```

Comment:
This line is unnecessary since a revision was published: https://hackage.haskell.org/package/singleton-bool-0.1.6/revisions/